packages/back-end/src/app.ts

```diff
@@ -429,6 +428,12 @@ app.get(
   experimentsController.getSnapshotWithDimension
 );
 app.post("/experiment/:id/snapshot", experimentsController.postSnapshot);
+
+app.get("/experiments/snapshots/", experimentsController.getSnapshots);
+app.post(
+  "/experiments/snapshots/scaled/",
+  experimentsController.postSnapshotsWithScaledImpactAnalysis
```

Comment:
Don't add trailing slashes to the routes.

Reply:
done, ty.